src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -26,4 +26,22 @@
     <PackageReference Include="System.Threading.ThreadPool" Version="4.4.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <CodeGenInput Include="..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\KnownHeaders.cs" />
+    <CodeGenInput Include="..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\FrameFeatureCollection.cs" />
+    <CodeGenOutput Include="Internal\Http\FrameHeaders.Generated.cs" />
+    <CodeGenOutput Include="Internal\Http\Frame.Generated.cs" />
+  </ItemGroup>
+
+  <Target Name="GenerateCode"
+          Inputs="@(CodeGenInput)"
+          Outputs="@(CodeGenOutput)"
+          BeforeTargets="CoreCompile">
+    <Exec Condition="!Exists('..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\obj\project.assets.json')"
```

Comment:
This is the equivalent of the lock file. You would just need to re-run restore to update dependencies. Or git clean.